src/main/scala/dissonance/model/activity/ActivityType.scala

```diff
@@ -17,4 +17,11 @@ object ActivityType {
     case 4     => Right(Custom)
     case other => Left(s"Unknown activity type ID: $other")
   }
+
```

Comment:
Something fun to try that would get rid of the kind of duplicate code in the Decoder and Encoder: Try to implement them both using this:
```scala
private val activityTypeCode: ActivityType => Int = {
  case Game      => 0
  case Streaming => 1
  case Listening => 2
  case Custom    => 4
}
```